src/content.js

```diff
@@ -53,8 +53,8 @@ function groupByRepo(events) {
 		return repos;
 	}, new Map());
 }
-function apply(options, insertionPoint) {
-	const holder = domifyEscape`<div class="ghcf-holder"><i>`;
+function apply(options) {
+	const holder = document.createDocumentFragment();
```

Comment:
> Also I remember that I used <i> to trick GitHub's own CSS to add a line above the first element:
Another reason to switch to the [template](https://github.com/bfred-it/github-clean-feed/issues/8)

Well, I'll try to restore it.